unable to compile cold package with arm-none-eabi 11

#### Expected Behavior
Compile the project
#### Actual Behavior:
After using conductor to make a new project I get errors when making the project.

#### Steps to reproduce:
I installed the toolchain through the Arch repos:
Versions:
arm-none-eabi-gcc 11.2.0-1
arm-none-eabi-binutils 2.37-1
arm-none-eabi-newlib 4.1.0-1
Make a new project with `pros conductor new-project /path/to/working/dir/`
When it tries to make the project, it gives me errors about the cold package as shown below.
It also gives the same error when using `pros make`

#### System information:
Operating System:
Arch Linux
5.14.12-arch1-1

PROS Version:
version 3.2.2

#### Screenshots/Output Dumps/Stack Traces
Creating cold package with libc,libm,libpros,okapilib [ERRORS]
/usr/lib/gcc/arm-none-eabi/11.2.0/../../../../arm-none-eabi/bin/ld: /usr/lib/gcc/arm-none-eabi/11.2.0/../../../../arm-none-eabi/lib/thumb/v7+fp/softfp/libstdc++.a(thread.o): in function `std::this_thread::__sleep_for(std::chrono::duration<long long, std::ratio<1ll, 1ll> >, std::chrono::duration<long long, std::ratio<1ll, 1000000000ll> >)':
/build/arm-none-eabi-gcc/src/gcc-11.2.0/libstdc++-v3/src/c++11/thread.cc:238: undefined reference to `sleep'
/usr/lib/gcc/arm-none-eabi/11.2.0/../../../../arm-none-eabi/bin/ld: /build/arm-none-eabi-gcc/src/gcc-11.2.0/libstdc++-v3/src/c++11/thread.cc:228: undefined reference to `usleep'
collect2: error: ld returned 1 exit status
ERROR - pros.cli.build:make - Failed to make project: Exit Code 2
make: *** [common.mk:229: bin/cold.package.elf] Error 1
Error: Failed to build

--- a/src/system/newlib_stubs.c
+++ b/src/system/newlib_stubs.c
@@ -37,7 +37,7 @@ int usleep( useconds_t period ) {
 		return 0;
 	}
 	uint64_t endTime = vexSystemHighResTimeGet() + period;
-	while(vexSystemHighResTimeGet() < endTime) asm("NOP");
+	while(vexSystemHighResTimeGet() < endTime) asm("YIELD");
 	return 0;
 }
 

